introduction_to_amazon_algorithms/jumpstart-foundation-models/text2text-generation-flan-t5-Batch-Transform.ipynb

```diff
@@ -0,0 +1,568 @@
+{
```

Comment:
<p>provide hyper-link for "Batch Transform" https://docs.aws.amazon.com/sagemaker/latest/dg/batch-transform.html, and also please explain it in a couple of lines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Sure</p>